Linked_list/linked_list.cpp

```diff
@@ -0,0 +1,151 @@
+#include <iostream>
+#include "linked_list.h";
+using namespace std;
+//creat a Node
+Node *CreatNode(Data data_in)
+{
+    Node *temp = new Node;
+   
+    temp->data = data_in;
+    temp->pnext = NULL;
+    return temp;
+
+};
+void init (List& lst)
```

Comment:
Let's respect the naming rule in your code. For example put the first character in Upper Case for function. --> "Init" instead of "init"